### ISSUE TITLE
Support for installation

### ISSUE THREAD

**Issue by appukuttan-shailesh** [solveignaess/fourspheremodel#2]
I work in the EBRAINS Model Validation Framework team, and we were keen to include validations for this model.
But we were having some difficulties in the setup.

I notice that this repo has been inactive for quite some time. Would there be someone who can assist with running the model in a Python virtual environment? The given conda installation instructions does not seem to work as expected currently. We saw that a docker based approach is mentioned [here](https://github.com/Neuroinflab/fourspheremodel), but we would prefer to avoid docker if possible (to retain our existing workflows).

**Comment by appukuttan-shailesh** [comment]
Tagging @solveignaess , @Neuroinflab

**Comment by torbjone** [comment]
Hi, thank you for your interest in running this :-)
I'm sure we can help you get this repo to work, but the reason it has been inactive for a while is that it was incorporated into the LFPy 2.0 software (https://doi.org/10.3389/fninf.2018.00092), and whenever we use the four-sphere head model, we use that implementation, see the class "FourSphereVolumeConductor" in this file: https://github.com/LFPy/LFPykit/blob/master/lfpykit/eegmegcalc.py 
Perhaps that would be a better starting point?

Best,
Torbjørn V Ness

**Comment by appukuttan-shailesh** [comment]
Hey @torbjone ... thanks for the quick response on this :-)
Great to hear that this is now merged into LFPy.

Our current motivtion is largely to do with the fact that this model has a curated entry in the EBRAINS Knowledge Graph:
https://search.kg.ebrains.eu/?category=Model&q=sphere%20four#05aa7c0d-b6b5-4b58-b938-f4086ff67ada
and we were trying to see if we could register formal validation tests (replicating those in the paper) for such curated models.

So with that in mind, we might be keen on running the original model itself (along with the other models which were used in the comparison). Could you take a look and see how one could get it running in a Python virtual environment? That would be super helpful.

In the meantime, if you do have examples of usage of the four-sphere head model via LFPy... especially those that reproduce the results from the [associated paper](https://www.frontiersin.org/articles/10.3389/fnhum.2017.00490/full), I could take a look at those as well.

**Comment by torbjone** [comment]
Most of the code is pure python, which should be unproblematic. The only potential problem is the Finite Element simulations using FEniCS, since this software has had some major updates since this code. Either, we will probably have to use an old version of FEniCS or update the code. Either way, I can look into it, but I will go on three weeks' vacation after this week, so I will not get the time to look at this before August.  

But regardless of this, I think we should modify the four-sphere entry at EBRAINS to link to the LFPy model entry (also on EBRAINS I'm quite sure) since this is what I would suggest that most people should use if they want to use the four-sphere model, can we do this?

**Comment by ccluri** [comment]
Hi all, the tagged version 1.0 is installed based on conda environment, but it's py27 and the links to the fenics library are broken. 

These issues were fixed by moving to docker and with the more recent updates by Kuba. I'm unable to look into this any time soon unfortuantely.

**Comment by appukuttan-shailesh** [comment]
Do you think if there already exists code/examples for reproducing Figs. 2, 3 and 4 of the [original paper](https://www.frontiersin.org/articles/10.3389/fnhum.2017.00490/full) using the LFPy version of the model? If so, I could give that a try. We are epecially keen on having the comparisons with other models.

**Comment by torbjone** [comment]
No, not really, because these figures also depend on the Finite Element implementation for validation, which was not incorporated into LFPy.

I'll get back to this in August :-)

**Comment by appukuttan-shailesh** [comment]
Thanks @torbjone 
Enjoy your break :-)
